lib/side-effect.js

```diff
@@ -77,9 +76,6 @@ export default function withSideEffect (reduceComponentsToState, handleStateChan
       }
     }
 
-    // Make UNSAFE_ compatible with version of React under 16.3
-    polyfill(SideEffect)
```

Comment:
@timneutkens I see that a couple tests still use `UNSAFE_` . . . do you want me to keep this line?

Reply:
Tests don't need the compat because we run tests on 16.4

Reply:
So removing it is perfect 👍